setup.py

```diff
@@ -20,15 +20,15 @@ def find_stub_files(name):
 
 dependencies = [
     "mypy>=0.950",
-    "django-stubs>=1.11.0",
+    "django-stubs>=1.12.0",
```

Comment:
I'm unsure how much I should bump these lower bound versions. Quite clearly these don't work in our CI, but in user projects they **may** work together?

If we want to be sure I suppose we should require mypy>=0.980 and django-stubs>=1.13.0.

Reply:
I think we can do whatever is easier for us to maintain :)

Reply:
Yeah after some thought, I would bump both of these.

Reply:
Bumped these version requirements.